### DISCUSSION TITLE
Add support for sentry-sdk

### DISCUSSION THREAD

**Discussion by hannseman** [cloudflare/workerd#1931] [Python Packages]
https://github.com/getsentry/sentry-python

**Comment by ryanking13** [comment]
sentry-sdk should be installable in Python workers. Given the broad scope of functionality covered by the sentry-sdk, let me close this as resolved, but we would appreciate it if you could report any issues you encounter to the [workers-py](https://github.com/cloudflare/workers-py) repository.
